task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 5)
+            {
+                int[] numberArray = new int[arrayLength];
+
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    Console.WriteLine($"Enter an array element № {i}");
+                    numberArray[i] = Convert.ToInt32(Console.ReadLine());
+                }               
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    bool isPrimeNumber = true;
```

Comment:
It is not the best way because now you create this variable on every iteration. It would be better to create this variable before the loop and then in this place you will only override value of `isPrimeNumber` for each iteration.
But you think in correct direction. It is right that you decided to set true in `isPrimeNumber` on each iteration. Nice!

Reply:
done